js/templates/modals/about/about.html

```diff
@@ -15,7 +15,7 @@ <h1 class="h3 txUp clrT2"><%= ob.polyT('tabMenuHeading') %></h1>
       <div class="contentBox padMd clrP clrBr clrSh3">
         <div class="flexVCent">
           <div><%= ob.brandingBoxT() %></div>
-          <h2 id=tabTitle class="h3 clrT txCtr flexExpand txUnl"></h2>
+          <h2 id="tabTitle" class="h3 clrT txCtr flexExpand txUnl"></h2>
```

Comment:
I just noticed this didn't have quotes around the ID value, and wanted to fix it. Didn't seem like it needed a separate PR.